### PR TITLE
Added Anagram.java

### PR DESCRIPTION
**Pull request by sj056**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**

* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Java files are placed inside an existing directory.
* [x] All filenames are in all uppercase characters with no spaces or dashes.
* [x] All functions and variable names follow Java naming conventions.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/Anagram.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+public class Anagram {
+
+	public static void main(String[] args) {
+		Scanner scan=new Scanner(System.in);
+		String a=scan.nextLine();
+		String b=scan.nextLine();
+		 boolean ana=true;
```
Comment (rbshealy) [inline strings/Anagram.java:8]: Fix indentation

File: strings/Anagram.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+public class Anagram {
+
+	public static void main(String[] args) {
+		Scanner scan=new Scanner(System.in);
```
Comment (rbshealy) [inline strings/Anagram.java:5]: add prompts for user input

File: strings/Anagram.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	      for(char c : a.toCharArray())
+	      {
+	          int i=(int)c;
+	          u[i]++;
+	      }
+	       for(char c : b.toCharArray())
+	      {
+	          int i=(int)c;
+	          v[i]++;
+	      }
```
Comment (rbshealy) [inline strings/Anagram.java:23]: What is going on here?

**Review by rbshealy — CHANGES_REQUESTED** [review]

File: strings/Anagram.java (hunk 3 above)
Comment (sj056) [reply strings/Anagram.java:23]: * Here i have initialized 2 arrays u and v first for string a and second for b .
* Converted the strings to lowercase. 
* Then alternatively, while converting the char of the string to int(we'll get the ASCII value of that character) then 
  incrementing that particular index of the string by 1(for eg : if char is 'a' then we'll increment the 97th(ASCII value of 
  a) index of the array).
*  We'll follow these steps for both the strings (hence 2 for loops are used) . 
* At the end we'll check if both the arrays are equal then the strings are anagram of each other else not.

**Pull request closed without merging**